README.md

```diff
@@ -24,7 +24,7 @@ See my tweet about this [here](https://twitter.com/reNotANumber/status/161499086
 
 ## Running:
 
-- See how to run on a watch device here: 
+- See how to run on a watch device [here](https://developer.android.com/training/wearables/get-started/debugging#wifi-debugging). 
```

Comment:
idk if this is the link you had in mind, but it looks like you meant to link to something here :)

Also, I filed https://issuetracker.google.com/issues/284186685 to update the Android docs with the extra Wear OS 3 details I found in https://www.google.com/url?q=https://www.reddit.com/r/WearOS/comments/z31ny0/psa_adb_to_wear_os_3_requires_extra_steps/&sa=D&source=buganizer&usg=AOvVaw0DcSee1P9cYdDgr5iFoQ5y